frontend/src/queries/nodes/DataTable/renderColumn.tsx

```diff
@@ -21,7 +24,11 @@ export function renderColumn(
     setQuery?: (query: DataTableNode) => void,
     context?: QueryContext
 ): JSX.Element | string {
-    if (key === 'event' && isEventsQuery(query.source)) {
+    if (value === loadingColumn) {
+        return <Spinner />
+    } else if (value === errorColumn) {
+        return <Tag color="red">Error</Tag>
```

Comment:
You're giving me extra work 😄 Why not `LemonTag`?

Reply:
I didn't find it 🤪 

<img width="635" alt="image" src="https://user-images.githubusercontent.com/53387/213453863-c0bc2460-e63f-4ed4-a870-ff8d956233ee.png">